airflow/exceptions.py

```diff
@@ -177,10 +177,6 @@ def serialize(self):
         )
 
 
-class UnmappableOperator(AirflowException):
```

Comment:
Oh, did not even know such existed :-D Learning something new every day... ah, now need to have un-seen...